docs/xks/operator-guide/eks.md

```diff
@@ -0,0 +1,111 @@
+---
+id: eks
+title: EKS
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Xenit Kubernetes Framework supports both AKS and EKS, though AKS is our main platform.
```

Comment:
I think we should avoid displaying preference like this. It only takes two customers on AWS and six months of operation for this not to be true, but it may be around for a long time and potentially scare off customers.